packages/kit/src/components/AccountSelector/NetworkSelectorTrigger/NetworkSelectorTriggerDApp.tsx

```
@@ -162,68 +160,56 @@
-    [handlePress, media.gtMd, network?.id, network?.name, triggerDisabled],
-  );
-
   return (
-    <Tooltip
-      shortcutKey={EShortcutEvents.NetworkSelector}
-      renderTrigger={trigger}
-      renderContent={intl.formatMessage({ id: ETranslations.global_network })}
-      placement="bottom"
-    />
+    <XStack
+      role="button"
```

Biome

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)